src/lib/ndpi_main.c

```diff
@@ -6020,6 +6004,19 @@ void ndpi_parse_packet_line_info(struct ndpi_detection_module_struct *ndpi_str,
 		      "Upgrade-Insecure-Requests: ", 27) == 0)) {
 	/* Just count. In the future, if needed, this if can be splited to parse these headers */
 	packet->http_num_headers++;
+      } else 
```

Comment:
@ragostino , while this "if/else list" seems fine, I am not sure that this code block that you moved should be put in it.\
Scenario: if we set `packet->content_line.ptr` around line 5900 (in that "else" block), we will need to check for these separators anyway, IMO.
Could you check, please?

Reply:
Good morning.
Yes, you are right! I edit and push a change.